tests/torch/test_model_transformer.py

```diff
@@ -505,3 +507,39 @@ def test_rebuild_graph_after_insert_transformation():
     transformed_model = model_transformer.transform(transformation_layout=transformation_layout)
     new_graph = transformed_model.nncf.get_graph()
     assert len(new_graph.get_all_nodes()) == len(graph.get_all_nodes()) + 1
+
+
+@pytest.mark.parametrize(
+    "target_type, node_name, input_port_id, ref_name",
+    (
+        (TargetType.OPERATOR_POST_HOOK, "/nncf_model_input_0", None, "/nncf_model_input_0|OUTPUT"),
+        (
+            TargetType.OPERATOR_PRE_HOOK,
+            "InsertionPointTestModel/linear_0",
+            0,
+            "InsertionPointTestModel/linear_0|INPUT0",
+        ),
+        (TargetType.OPERATION_WITH_WEIGHTS, "InsertionPointTestModel/NNCFConv2d[conv1]/conv2d_0", None, None),
+    ),
+)
+def test_quantizer_insertion_transformations(target_type, node_name, input_port_id, ref_name):
+    model = NNCFNetwork(InsertionPointTestModel(), [ModelInputInfo([1, 1, 10, 10])])
+    model_transformer = PTModelTransformer(model)
+
+    target_point = PTTargetPoint(target_type, node_name, input_port_id=input_port_id)
+    command = PTQuantizerInsertionCommand(target_point, BaseOp(lambda x: x))
+
+    transformation_layout = PTTransformationLayout()
+    transformation_layout.register(command)
+    transformed_model = model_transformer.transform(transformation_layout)
+
+    assert hasattr(transformed_model.nncf, "external_quantizers")
```

Comment:
BTW, this should reference the corresponding constant. I did a quick search on "external_quantizers" in the codebase and noticed other places where you use it verbatim instead of referring to the constant (e.g. in strip functionality). Replace those usage with reference to the constant as well.

Reply:
Good point. Updated. 
And added 'NNCFNetworkInterface:is_compression_module_registered' method to check that extra module are registered without raising Runtime exception.